GANDLF/inference_manager.py

```diff
@@ -22,6 +22,8 @@ def InferenceManager(dataframe, outputDir, parameters, device):
 
     n_folds = parameters["nested_training"]["validation"]
 
+    if "second_output_dir" in parameters:
+        outputDir = outputDir + "," + parameters["second_output_dir"]
```

Comment:
To clarify, gandlf_run and the training/inference loops treat the model dir as the output dir. This is problematic in the case of a model embedded in a docker image -- if you run training on top of that model and do the output to a directory inside the container, it won't persist. Likewise for inference -- we need a way to place the results not in the model directory itself, but in a separate location (i.e. a writable location that was mounted into the container, which the MLCube runners and MedPerf do for us). But those runners don't do auto-mounting unless the the output location is given from the mlcube.yaml file as a parameter to the command. 

Because of this, and not wanting to change the existing (expected) functionality for how the model directory is also used for output, I created this as a new option (which happens to also be convenient for people who are, say, trying different techniques on iterative training and want to back up each version to a different location). Also, concatenating this to the existing outputDir would cause inference to try and treat the second output directory as a separate model. Not what we want.

Reply:
Sounds good to me. Just made a commit that replaces it.